econf/tests/basics.rs

```diff
@@ -489,6 +491,30 @@ fn duration() {
     assert_eq!(a.d3, Duration::from_secs(3600));
 }
 
+#[derive(LoadEnv)]
+struct Paths {
+    p1: PathBuf,
+    p2: PathBuf,
+    p3: PathBuf,
+}
+
+#[test]
+fn path_buf() {
+    std::env::set_var("PATHS_P2", "other/path/to/file.toml");
+    std::env::set_var("PATHS_P3", "data.db");
+
+    let a = Paths {
+        p1: PathBuf::from_str("path/to/dir/").unwrap(),
+        p2: PathBuf::from_str("path/to/file.toml").unwrap(),
+        p3: PathBuf::from_str("file.rs").unwrap(),
```

Comment:
I think this is a matter of preference but we can also use

```suggestion
        p3: "file.rs".parse().unwrap(),
```

no need to import FromStr in this case

Reply:
Absolutely :D Done!